docs/README.md

```diff
@@ -108,11 +108,11 @@ When renaming or moving a Cozy-UI component, it may cause a breaking change. In
 
 If you want to add a new icon to cozy-ui, you must follow these steps:
 
-* First verify that the SVG doesn't have any `fill` or `fill-opacity` properties. Remove them if necessary
+* [For icon only] Verify that the SVG doesn't have any `fill` or `fill-opacity` properties. Remove them if necessary
```

Comment:
If you SVG file is an icon (not an illustration), verify that the file doesn't have any `fill` or `fill-opacity` properties. Remove them if necessary

je trouve ça plus explicite...